### PR TITLE
[Communication] - Phone Numbers - Search tests disabled for temporary issue

### PR DESCRIPTION
**Pull request by jbeauregardb**

### CODE REVIEW

**Review by minnieliu — APPROVED** [review]

File: sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClientIntegrationTest.java [hunk 1]
```diff
@@ -110,6 +113,9 @@ public void beginSearchAvailablePhoneNumbers(HttpClient httpClient) {
 
     @ParameterizedTest
     @MethodSource("com.azure.core.test.TestBase#getHttpClients")
+    @DisabledIfEnvironmentVariable(
+        named = "SKIP_LIVE_TEST",
+        matches = "(?i)(true)")
     public void beginSearchAvailablePhoneNumbersWithoutOptions(HttpClient httpClient) {
```
Comment (lsundaralingam) [inline sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClientIntegrationTest.java:119]: Do we need to disable this test for sync as well?

**Review by lsundaralingam — APPROVED** [review]

File: sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClientIntegrationTest.java (hunk 1 above)
Comment (jbeauregardb) [reply sdk/communication/azure-communication-phonenumbers/src/test/java/com/azure/communication/phonenumbers/PhoneNumbersAsyncClientIntegrationTest.java:119]: This test doesn't exist in the sync test file

**Pull request merged**